src/features/game/expansion/placeable/RemovePlaceableModal.tsx

```diff
@@ -49,6 +56,58 @@ export const RemovePlaceableModal: React.FC<Props> = ({
     onClose();
   };
 
+  const getCanRemove = () => {
```

Comment:
Is it possible to extract this functionality into a separate file? 

We have some critical business logic here that we can separate from the UI layer. This will make it easier for testing 🙏

Reply:
I separated that method into a new file and added unit tests